projects/plugins/jetpack/modules/memberships/class-jetpack-memberships.php

```diff
@@ -431,12 +431,26 @@ public static function get_connected_account_id() {
 	}
 
 	/**
-	 * Get the newsletter access level
+	 * Get the post access level
 	 *
-	 * @return string|void
+	 * @return string the actual post access level (see projects/plugins/jetpack/extensions/blocks/subscriptions/settings.js for the values).
 	 */
-	public static function get_newsletter_access_level() {
-		return get_post_meta( get_the_ID(), self::$newsletter_access_level_meta_name, true );
+	public static function get_post_access_level() {
```

Comment:
Renamed for consistency.

Reply:
Nice! But also quite hard to read when reviewing (GitHub UI can be tricky)